presto-product-tests/bin/run_on_docker.sh

```diff
@@ -35,6 +39,18 @@ function stop_unnecessary_hadoop_services() {
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop zookeeper
 }
 
+function start_mapr_httpfs_service() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  docker exec ${HADOOP_MASTER_CONTAINER} chown -R mapr ${MAPR_HOME}/httpfs
+  docker exec ${HADOOP_MASTER_CONTAINER} maprcli node services -name httpfs -action start -nodes ${HADOOP_HOST_NAME}
```

Comment:
why is that done from this script? why doesn't the image do this on startup (in, say, bootstrap.sh?)